lib/common_components/custom_input_field.dart

```diff
@@ -77,6 +79,11 @@ class CustomInputField extends StatelessWidget {
           children: <Widget>[
             Expanded(
               child: TextField(
+                autocorrect: false, // textfield autocorrect off
+                enableSuggestions: false,
+                keyboardType: textInputType ??
+                    TextInputType
+                        .visiblePassword, // Tweak, if the device's keyboard's autocorrect is on
```

Comment:
you can actually go with `TextInputType.text` as default one. This `TextInputType.visiblePassword` is optimize for passwords that are visible to the user.

Reply:
Well actually that proved to be one of the ways to disable autocorrect. I have intentionally used it. Do I need to change it anyways?

Reply:
ahhh, cool. that works in that case. Perfect.